ports/rp2/CMakeLists.txt

```diff
@@ -271,6 +304,10 @@ if (MICROPY_PY_NETWORK_CYW43)
         cyw43_driver_picow
         cmsis_core
     )
+    target_include_directories(${MICROPY_TARGET} PRIVATE
```

Comment:
Should be above in MICROPY_PY_BLUETOOTH_CYW43?

Reply:
I think it's a wifi issue with the firmware inclusion. I'll leave it where it is.

/home/peterh/source/pico/micropython_latest/lib/cyw43-driver/src/cyw43_config.h:64:45: fatal error: firmware/wb43439A0_7_95_49_00_combined.h: No such file or directory

   64 | #define CYW43_CHIPSET_FIRMWARE_INCLUDE_FILE "firmware/wb43439A0_7_95_49_00_combined.h"